Azure/communication-ui-library-ios
# [Task] Send read receipt when scrolling ends

## Purpose

* Send read receipt of the last remote message on screen when scroll ends.

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull request checklist

This PR has considered:

## How to Test


* Open a chat thread
* Should send out a read receipt for the last remote message on screen 
* Scroll down to another remote message
* Another read receipt should be sending out



## Other Information

We can set a breakpoint on this line `dispatch(.participantsAction(.sendReadReceiptTriggered(messageId: messageId)))` to observe the behaviour
